fvm/src/gas/price_list.rs

```diff
@@ -241,9 +238,14 @@ lazy_static! {
         .copied()
         .collect(),
 
-        // TODO(#1277): Implement this first before benchmarking.
         // TODO(#1384): Reprice
```

Comment:
I didn't actually do what this issue asks for (re-benchmark), so I didn't drop this TODO.

I think we can probably close the issue and drop the TODO, though, I have a hard time imagining we'll ever prioritize this?